MCC_cross_sections/He/README.md

```diff
@@ -0,0 +1,15 @@
+The origins of the included cross-sections are as follows:
+ - electron neutral isotropic scattering
+    S. F. Biagi's FORTRAN code MAGBOLTZ, version 7.1 (June 2004)
+ - excitation 1
+    S. F. Biagi's FORTRAN code MAGBOLTZ, version 7.1 (June 2004)
+ - excitation 2
+    S. F. Biagi's FORTRAN code MAGBOLTZ, version 7.1 (June 2004)
+ - ionization
+    S. F. Biagi's FORTRAN code MAGBOLTZ, version 7.1 (June 2004)
+ - ion neutral isotropic scattering
+    A. V. Phelps database (available on LxCat)
+ - ion back scattering
+    A. V. Phelps database (available on LxCat)
+ - ion-neutral charge exchange
+    sigma_ct = sigma_bs + 0.5 * sigma_iso
```

Comment:
Just a small question:

Shall we add a comment here or in the other README on the columns (name of column & unit)?

Reply:
Great point, thanks!